custom-components/react-carousel/carousel.jsx

```diff
@@ -0,0 +1,110 @@
+const newId = (() => {
```

Comment:
[Optional]
Was there any particular reason to remove all semicolons?
Unless JSX solves ASI issues I'd rather keep them. More info with some examples about ASI edge cases [here](https://github.com/airbnb/javascript#semicolons).

Reply:
No idea really, i've been doing some React tutorials lately and realized they are mostly left out, but very inconsistently. I tried to consistently avoid them, I'm aware of the edge cases but we don't have any here, and the edge cases mostly happen with weird/poor code, so. From what I read regarding it many consider airbnb as outdated.

 Should I add them? I'm kind of new to ES6, also why i tried to do this, so excuse my doubts here.

Reply:
If you ask me, i like it better without semicolons